codegen-core/src/main/kotlin/io/vrap/rmf/codegen/types/LanguageBaseTypes.kt

```diff
@@ -1,7 +1,7 @@
 package io.vrap.rmf.codegen.types
 
 abstract class LanguageBaseTypes(
-
+        val anyType: VrapType,
```

Comment:
I need a differentiation between the `any` and `object` type. Cause `any` will be treated as a scalar where as `object` as a class. For Java it may not be problematic as everything is an object. But for PHP it is.

Reply:
Thanks for the explanation @jenschude 👍